pkg/agent/openflow/multicluster.go

```diff
@@ -184,3 +186,22 @@ func (f *featureMulticluster) snatConntrackFlows(serviceCIDR net.IPNet, localGat
 	)
 	return flows
 }
+
+func (f *featureMulticluster) l3FwdFlowToPodViaTun(
+	localGatewayMAC net.HardwareAddr,
+	podIP net.IP,
+	tunnelPeer net.IP) binding.Flow {
+	ipProtocol := getIPProtocol(podIP)
+	// This generates the flow to forward cross-cluster request packets based
+	// on Pod IP.
+	return L3ForwardingTable.ofTable.BuildFlow(priorityNormal).
```

Comment:
I was asking why it must use another rule to process this packet instead of the existing rule installed for in-cluster traffic. I now got it's because the VNI. I guess this should still use higher priority, otherwise packets may hit the other rule which doesn't encapsulate packet.

Reply:
You are right, updated it to `priorityHigh`